starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +14,41 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(words) {
+  var longest=[];
+  for (let i = 0; i < words.length; i += 1) {
+    if (longest.length < words[i].length) {
+      return longest = words[i];
+    } else if (longest.length == words[i].length){
+      return longest = words[i];
+    }
+  }
+}
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumArray(numbers) {
+  let count = 0;
+  for(let i = 0; i < numbers.length; i += 1){
+    count += numbers[i];
+  }
+  return count;
+}
+
 // Calculate the Average
 
 var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(numbersAvg) {
```

Comment:
Considerar a possibilidade de array vazio: `numbersAvg.length = 0` e também se nenhum valor for passado ao argumento: `numbersAvg = undefined`